shared/common-adapters/confirm-modal/index.js

```diff
@@ -38,9 +40,9 @@ class _ConfirmModal extends React.Component<Props> {
               <Icon
                 boxStyle={styles.icon}
                 color={Styles.globalColors.black_50}
-                fontSize={Styles.isMobile ? 64 : 48}
+                fontSize={Shared.typeToFontSize('Big')}
```

Comment:
Sorry, I meant this would be `sizeType="Huge"`, and in case of `Big`, `sizeType="Big"`. Although it seems we don't have a conclusion if `Huge` should be added at this time, so to make it huge this would still needs to be what you had before `fontSize={Styles.isMobile ? 64 : 48}`.

Reply:
:white_check_mark: